plugins/newsdownloader.koplugin/main.lua

```diff
@@ -25,10 +25,11 @@ local initialized = false
 local wifi_enabled_before_action = true
 local feed_config_file_name = "feed_config.lua"
 local news_downloader_config_file = "news_downloader_settings.lua"
-local config_key_custom_dl_dir = "custom_dl_dir";
+local config_key_custom_dl_dir = "custom_dl_dir"
 local file_extension = ".epub"
 local news_download_dir_name = "news"
 local news_download_dir_path, feed_config_path
+local never_download_images_config_name = "never_download_images_config_name"
```

Comment:
I find this unnecessarily complex. I don't follow the reasoning behind not to just using `readSetting("never_download_images")`, but setting that aside:
```suggestion
local never_download_images_config_name = "never_download_images"
```